app/pub_sub/subscribers/site_activity.rb

```diff
@@ -0,0 +1,33 @@
+module Subscribers
+  class SiteActivity < ::Subscribers::Base
+    def site_created(event)
+      return if event.payload[:subject].nil? || event.payload[:subject].new_record?
```

Comment:
Sure, that seems sensible. I think we can manage to make simpler subscribers (almost logic-less) by triggering publishers after certain events. That may add some complexity to the action's workflow but I'd say it's totally worth it.